src/pv.rs

```diff
@@ -61,6 +62,62 @@ impl PvPacket {
             private: std::ptr::null_mut(),
         }
     }
+
+    // check whether the new header can be attached to payload from the start of packet
+    fn is_addable(header_length: usize, packet: &PvPacket) -> bool {
```

Comment:
"Prepend" or "Attach" instead of "Add"

Reply:
the name is changed with is_attachable()